steps/property-inspection/damage.tsx

```diff
@@ -0,0 +1,121 @@
+import { FieldsetLegend } from "lbh-frontend-react/components";
+import React from "react";
+import {
+  ComponentDatabaseMap,
+  ComponentValue,
+  ComponentWrapper,
+  DynamicComponent
+} from "remultiform/component-wrapper";
+
+import { ImageInput } from "../../components/ImageInput";
+import { makeSubmit } from "../../components/makeSubmit";
+import ProcessStepDefinition from "../../components/ProcessStepDefinition";
+import { RadioButtons } from "../../components/RadioButtons";
+import { TextArea } from "../../components/TextArea";
+import DatabaseSchema from "../../storage/DatabaseSchema";
+import processRef from "../../storage/processRef";
+
+import PageSlugs, { hrefForSlug } from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const step: ProcessStepDefinition = {
+  title: PageTitles.Damage,
+  heading: "Are there any signs of damage caused to the property?",
+  step: {
+    slug: PageSlugs.Damage,
+    nextSlug: PageSlugs.Submit,
+    Submit: makeSubmit({
+      href: hrefForSlug(PageSlugs.Submit),
+      value: "Save and continue"
+    }),
+    componentWrappers: [
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "has-damage",
+          Component: RadioButtons,
+          props: {
+            name: "has-damage",
+            legend: (
+              <FieldsetLegend>
+                Are there any signs of damage caused to the property?
+              </FieldsetLegend>
+            ) as React.ReactNode,
+            radios: [
+              {
+                label: "Yes",
+                value: "yes"
+              },
+              {
+                label: "No",
+                value: "no"
+              }
+            ]
+          },
+          defaultValue: "",
+          emptyValue: "",
+          databaseMap: new ComponentDatabaseMap<DatabaseSchema, "property">({
+            storeName: "property",
+            key: processRef,
+            property: ["damage", "hasDamage"]
+          })
+        })
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "damage-images",
+          Component: ImageInput,
+          props: {
+            label: "Take photos of damage",
+            name: "damage-images",
+            hintText: "You can take up to 5 different photos" as
```

Comment:
Is this limit ok? I can imagine them needing more if there's a lot of damage. Check with Gill?

Reply:
Spoke to Gill about this, she seemed quite fine with 3 for everything. Figured the UR session on multiple images would root out whether they needed to upload more or not.